extension_config.cmake

```diff
@@ -0,0 +1,11 @@
+# This file is included by DuckDB's build system. It specifies which extension to load
+
+# Extension from this repo
+duckdb_extension_load(athena
+    SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}
+    LOAD_TESTS
+    LINKED_LIBS "../../cargo/build/x86_64-unknown-linux-gnu/release/libduckdb_athena.a"
```

Comment:
Finally got it to build on Linux for me, but this will need to be changed to identify the proper build architecture/platform. 

I wasn't able to actually get a query to work, still digging into that and also trying to get it to build on my mac.

Reply:
That makes sense.  I wonder if changing LINKED_LIBS to `"${CMAKE_CURRENT_BINARY_DIR}/libduckdb_athena.a"` would work.

The query I got to work looked something like:
```$ AWS_REGION=us-west-2 build/release/duckdb -unsigned
v1.1.1 af39bd0dcf
Enter ".help" for usage hints.
D load 'build/release/extension/athena/athena.duckdb_extension';
D select * from athena_scan('example_csv', 's3://aws-athena-query-results-************-us-west-2/');
```